settings.gradle

```diff
@@ -90,8 +90,12 @@ project(":code/complex-number-calculator").name = "complex-number-calculator-mod
 // Купцова Алеся
 // Кучков Иван
 // Лапин Артем
```

Comment:
One comment: Pls rename it to your project name, for example: Quadratic equation

Reply:
Updated